wizard.sh crashes on non systemd-based system

```
[ec2-user@ip-172-31-27-163 crowdsec]$ uname -a
Linux ip-172-31-27-163 4.14.177-107.254.amzn1.x86_64 #1 SMP Thu May 7 18:30:14 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```
System is amazon linux, latest version. The real culprit seems to be the empty configuration file.

The use of `wizard.sh -i` with all default entries result with:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xd87374]

goroutine 1 [running]:
main.initConfig()
	/data/work/crowdsec/cmd/crowdsec-cli/main.go:56 +0x304
github.com/spf13/cobra.(*Command).preRun(0xc0003fe580)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:868 +0x49
github.com/spf13/cobra.(*Command).execute(0xc0003fe580, 0xc0003fc540, 0x3, 0x3, 0xc0003fe580, 0xc0003fc540)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:804 +0x145
github.com/spf13/cobra.(*Command).ExecuteC(0xc000224580, 0xc0002dbf28, 0x1, 0x1)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:943 +0x317
github.com/spf13/cobra.(*Command).Execute(...)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:883
main.main()
	/data/work/crowdsec/cmd/crowdsec-cli/main.go:139 +0x93d
[26/05/2020:06:43:06][ERR] crowdsec_wizard: fail to install collection crowdsec/whitelists
[26/05/2020:06:43:08][INF] crowdsec_wizard: Installing patterns
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xd87374]

goroutine 1 [running]:
main.initConfig()
	/data/work/crowdsec/cmd/crowdsec-cli/main.go:56 +0x304
github.com/spf13/cobra.(*Command).preRun(0xc0003fedc0)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:868 +0x49
github.com/spf13/cobra.(*Command).execute(0xc0003fedc0, 0x173b730, 0x0, 0x0, 0xc0003fedc0, 0x173b730)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:804 +0x145
github.com/spf13/cobra.(*Command).ExecuteC(0xc000222580, 0xc0002d9f28, 0x1, 0x1)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:943 +0x317
github.com/spf13/cobra.(*Command).Execute(...)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:883
main.main()
	/data/work/crowdsec/cmd/crowdsec-cli/main.go:139 +0x93d
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xd87374]

goroutine 1 [running]:
main.initConfig()
	/data/work/crowdsec/cmd/crowdsec-cli/main.go:56 +0x304
github.com/spf13/cobra.(*Command).preRun(0xc000401340)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:868 +0x49
github.com/spf13/cobra.(*Command).execute(0xc000401340, 0x173b730, 0x0, 0x0, 0xc000401340, 0x173b730)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:804 +0x145
github.com/spf13/cobra.(*Command).ExecuteC(0xc000222580, 0xc0002dbf28, 0x1, 0x1)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:943 +0x317
github.com/spf13/cobra.(*Command).Execute(...)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:883
main.main()
	/data/work/crowdsec/cmd/crowdsec-cli/main.go:139 +0x93d
[26/05/2020:06:43:08][ERR] crowdsec_wizard: unable to register, skipping crowdsec api registration
[26/05/2020:06:43:08][INF] crowdsec_wizard: Crowdsec api registered
./wizard.sh: ligne462: systemctl : commande introuvable
[26/05/2020:06:43:08][ERR] crowdsec_wizard: unable to start crowdsec. exiting
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xd87374]

goroutine 1 [running]:
main.initConfig()
	/data/work/crowdsec/cmd/crowdsec-cli/main.go:56 +0x304
github.com/spf13/cobra.(*Command).preRun(0xc0003ff600)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:868 +0x49
github.com/spf13/cobra.(*Command).execute(0xc0003ff600, 0x173b730, 0x0, 0x0, 0xc0003ff600, 0x173b730)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:804 +0x145
github.com/spf13/cobra.(*Command).ExecuteC(0xc000222580, 0xc0002d9f28, 0x1, 0x1)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:943 +0x317
github.com/spf13/cobra.(*Command).Execute(...)
	/home/ec2-user/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:883
main.main()
	/data/work/crowdsec/cmd/crowdsec-cli/main.go:139 +0x93d
```

--- a/cmd/crowdsec-cli/main.go
+++ b/cmd/crowdsec-cli/main.go
@@ -53,6 +53,9 @@ func initConfig() {
 	/*read config*/
 	config.InstallFolder = filepath.Clean(csConfig.ConfigFolder)
 	config.HubFolder = filepath.Clean(config.configFolder + "/hub/")
+	if csConfig.OutputConfig == nil {
+		log.Fatalf("Missing backend plugin configuration in %s", config.ConfigFilePath)
+	}
 	config.BackendPluginFolder = filepath.Clean(csConfig.OutputConfig.BackendFolder)
 	config.DataFolder = filepath.Clean(csConfig.DataFolder)
 	//

--- a/pkg/leakybucket/manager.go
+++ b/pkg/leakybucket/manager.go
@@ -168,8 +168,8 @@ func LoadBuckets(files []string, dataFolder string) ([]BucketFactory, chan types
 			g.ret = response
 			err = LoadBucket(&g, dataFolder)
 			if err != nil {
-				log.Errorf("Failed to load bucket : %v", err)
-				return nil, nil, fmt.Errorf("loadBucket failed : %v", err)
+				log.Errorf("Failed to load bucket %s : %v", g.Name, err)
+				return nil, nil, fmt.Errorf("loading of %s failed : %v", g.Name, err)
 			}
 			ret = append(ret, g)
 		}

